politeiawww/config.go

```diff
@@ -84,6 +85,12 @@ var (
 	defaultRPCCertFile   = filepath.Join(sharedconfig.DefaultHomeDir, "rpc.cert")
 	defaultCookieKeyFile = filepath.Join(sharedconfig.DefaultHomeDir, "cookie.key")
 	defaultLogDir        = filepath.Join(sharedconfig.DefaultHomeDir, defaultLogDirname)
+
+	// Default start date to start pulling code statistics if none specified
+	defaultCodeStatStart = time.Date(2020, time.March, 0, 0, 0, 0, 0, time.UTC)
```

Comment:
This should be relative to the current date (ex. previous 6 months) instead of being an absolute date.